lib/workos/configuration.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+# typed: false
+
+module WorkOS
+  # Configuration class sets config initializer
+  class Configuration
+    attr_accessor :timeout, :key
+
+    def initialize
+      @timeout = timeout || 60
+      @key = key
+    end
+
+    def key!
+      key || raise('WorkOS.config.key not set')
```

Comment:
Let's omit the explicit parens for `raise`, and also wrap the programmatic identifiers with backticks like we do for other errors/warnings:

```suggestion
      key or raise 'WorkOS.config.key not set'
```